racetime/views/category.py

```diff
@@ -117,6 +117,31 @@ def get_json_data(self):
         return self.get_object().dump_json_data()
 
 
+class CategoryListData(generic.View):
+    def get(self, request):
+        age = settings.RT_CACHE_TIMEOUT.get('CategoryListData', 0)
+        content = cache.get_or_set(
+            'categories/data',
+            self.get_json_data,
+            age,
+        )
+        resp = http.HttpResponse(
+            content=content,
+            content_type='application/json',
+        )
+        if age:
+            resp['Cache-Control'] = 'public, max-age=%d, must-revalidate' % age
+        resp['X-Date-Exact'] = timezone.now().isoformat()
+        return resp
+
+    def get_json_data(self):
+        return json.dumps({
+            'categories': [
+                c.api_dict_summary() for c in Category.model.objects.all()
```

Comment:
Can you change this to `Category.model.objects.filter(active=True)` please?

Might want to stick an `.order_by('name')` too?